src/Lykke.Job.BlockchainMonitoring.AzureRepositories/ActiveOperations/ActiceCashoutEntity.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using Lykke.AzureStorage.Tables;
+
+namespace Lykke.Job.BlockchainMonitoring.AzureRepositories.ActiveOperations
+{
+    internal class ActiceCashoutEntity:AzureTableEntity
```

Comment:
Actice is a mistype, should be Active here and in file name, isn't it?
```suggestion
    internal class ActiveCashoutEntity:AzureTableEntity
```

Reply:
fixed - but pr summary shows old code.. please take a look at source code